pkg/acsengine/defaults.go

```diff
@@ -310,7 +310,9 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 			o.KubernetesConfig.EtcdVersion = DefaultEtcdVersion
 		}
 		if a.HasWindows() {
-			o.KubernetesConfig.NetworkPolicy = DefaultNetworkPolicyWindows
+			if o.KubernetesConfig.NetworkPolicy == "" {
+				o.KubernetesConfig.NetworkPolicy = DefaultNetworkPolicyWindows
```

Comment:
Why does Windows need to default to "none"? Originally it was default to "none" because Windows doesn't support azure cni.

Reply:
Addressed in const.go by setting DefaultNetworkPolicyWindows to "azure".